Aspire Budgeting/Facilities/GoogleSheetsManager.swift

```diff
@@ -260,8 +306,13 @@ final class GoogleSheetsManager: ObservableObject {
     ticket = sheetsService.executeQuery(appendQuery, completionHandler: { (_, data, error) in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/25c2f5d2b54fb8275ada47af8d5a461ce85fbfd7/Aspire Budgeting/Facilities/GoogleSheetsManager.swift#L306"></span>Unused parameter “data” in a closure should be replaced with _.<br />
<code>unused_closure_parameter</code> <code>GoogleSheetsManager.swift:306</code></td>
    </tr>
  </tbody>
</table>